WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -929,6 +934,20 @@ def setJobJDL(self, jobID, jdl=None, originalJDL=None):
     return result
 
 #############################################################################
+  def __compressJDL(self, jdl):
+    """Return compressed JDL string."""
+    if not self.compressJDLs:
+      return jdl
+    return zlib.compress(jdl, -1).encode('base64')
+
+  def __extractJDL(self, compressedJDL):
+    """Return decompressed JDL string."""
+    # the starting bracket is guaranteeed by JobManager.submitJob
+    # we need the check to be backward compatible
+    if not self.compressJDLs or compressedJDL.startswith('['):
```

Comment:
Actually this should be 
```suggestion
    if compressedJDL.startswith('['):
```
So that one can turn off compressJDLs after it was once enabled